Makefile

```
@@ -75,10 +76,24 @@ local: ## - Build local binary for local environment (bin/fleet-server)
 	go build $(if $(DEV),-tags="dev",) -gcflags="${GCFLAGS}" -ldflags="${LDFLAGS}" -o ./bin/fleet-server .
 	@printf "${CMD_COLOR_ON} Binaries in ./bin/\n${CMD_COLOR_OFF}"
 
+
+.PHONY: cover-e2e-binaries
+cover-e2e-binaries: ## - Build binaries for the test-e2e target with the go 1.20+ cover flag
+	SNAPSHOT=true $(MAKE) cover-$(shell go env GOOS)/$(shell go env GOARCH) cover-linux/$(shell go env GOARCH)
```

In several places we are indicating amd64 as the architecture, should we do it here too to avoid problems in ARM laptops?
```suggestion
	SNAPSHOT=true $(MAKE) cover-$(shell go env GOOS)/$(shell go env GOARCH) cover-linux/amd64
```